boxsdk/pagination/page.py

```diff
@@ -38,7 +39,7 @@ def _translator(self):
 
         :rtype:   :class:`Translator`
         """
-        return self._session.translator
+        return Translator()
```

Comment:
Why can't we use the session's translator? Is it okay to create a new one every time here? Maybe a comment could help?